gstools/variogram/estimator.pyx

```diff
@@ -150,18 +150,22 @@ def unstructured(
     cdef int i_max = bin_edges.shape[0] - 1
     cdef int j_max = x.shape[0] - 1
     cdef int k_max = x.shape[0]
+    cdef int f_max = f.shape[0]
 
     cdef vector[double] variogram = vector[double](len(bin_edges)-1, 0.0)
     cdef vector[long] counts = vector[long](len(bin_edges)-1, 0)
-    cdef int i, j, k
+    cdef int i, j, k, m
```

Comment:
You skipped my favorite letter!! :-p

Reply:
See here : https://www.python.org/dev/peps/pep-0008/#names-to-avoid